src/HeaderRow.tsx

```diff
@@ -25,21 +26,22 @@ export interface HeaderRowProps<R, SR, K extends React.Key> extends SharedDataGr
 }
 
 const headerRow = css`
-  contain: strict;
-  contain: size layout style paint;
-  display: grid;
-  grid-template-columns: var(--rdg-template-columns);
-  grid-template-rows: var(--rdg-header-row-height);
-  height: var(--rdg-header-row-height); /* needed on Firefox */
+  display: contents;
   line-height: var(--rdg-header-row-height);
-  width: var(--rdg-row-width);
-  position: sticky;
-  top: 0;
   background-color: var(--rdg-header-background-color);
   font-weight: bold;
-  z-index: 3;
   outline: none;
 
+  > .${cell} {
+    z-index: 3;
+    position: sticky;
+    top: 0;
+  }
+
+  > .${cellFrozen} {
+    z-index: 4;
+  }
+
   &[aria-selected='true'] {
     box-shadow: inset 0 0 0 2px var(--rdg-selection-color);
```

Comment:
I don't think this still works

Reply:
I don't think it ever worked. We never set `aria-selected` on the header row 🤔